common/db/src/create_db.rs

```diff
@@ -22,7 +22,7 @@ pub fn db_key(db_dst: &'static [u8], item_dst: &'static [u8], key: impl AsRef<[u
 ///
 /// # Example
 ///
-/// ```
+/// ```no_run
```

Comment:
Why not simply correct the missing import?

Reply:
I believe the doc test is compiled in the current scope, but those types I use aren't going to be available, as is the case with the `bincode` dependency - so correcting the missing import would just lead to more errors :sweat_smile:

Reply:
sorry if I was spamming CI :blush:

Reply:
Ha, don't worry. We're not on any paid plan.